app/src/main/res/values/colors.xml

```diff
@@ -20,6 +20,7 @@
 
 
     <!-- Content backgrounds -->
+    <color name="bg">#FFFFFF</color>
```

Comment:
Maybe that name should be more specific? What kind of bg is that? Where is it used?

Reply:
set background color to transparent